src/routes/login/+page.server.ts

```diff
@@ -32,10 +32,10 @@ export const actions: Actions = {
             return setError(form, 'password', 'Incorrect login or password');
         }
 
-        event.cookies.set('access_token', authResponse.access_token, { path: '/', sameSite: 'strict' })
+        event.cookies.set('access_token', authResponse.access_token, { path: '/', sameSite: 'strict', httpOnly: false })
```

Comment:
We should consider setting httpOnly back to true.
The cookies without httpOnly flag are vulnerable to attacks, and the token may be leaked as it is accessible by using JS in the browser.